### DISCUSSION TITLE
POLL: What coins are you most interested in seeing support for in future releases of UW?

### DISCUSSION THREAD

**Discussion by 3point14159** [horizontalsystems/unstoppable-wallet-android#5826] [Polls]
Instead of making a separate issue for each request for coin support, I thought it might be more helpful to run a poll on it to help the devs prioritize what the market is seeking. Each of the blockchains suggested are BIP39 compliant, and should be easy to add to the kit. I will monitor this post and add more to the list as other's add suggestions in the replies. A big thanks to the dev team for all considerations they're able to implement.

**Comment by OneNewSmartIdea** [comment]
Richard Heart projects, an Eth ERC20 token:
HEX [HEX] https://hex.com/

A not launched yet EVM blockchain:
Pulsechain  [PLS] https://pulsechain.com/

And the DEX on the blockchain:
PulseX  [PLSX] https://pulsex.com/   ( Uniswap v2 fork )

**Comment by serrq** [comment]


**Reply by 3point14159** [reply]
It would be nice to see both ARRR and OXEN in Unstoppable Wallet since they are both XMR forks. Would be cool to see XMR as well.

**Reply by serrq** [reply]
For some reason StealthEX has removed ARRR from its swap system list. 

Where I can swapping ARRR in non-surveillance based countries cex/dex?

**Comment by serrq** [comment]
Please adding ALGO (Algorand).

ARRR is currently out of my radar. No widely accepted/swapped.

**Comment by 3point14159** [comment]
Just to throw my hat in for a blockchain that is actually moving things in real-space; would love to see LBRY's LBC supported.

It could be a good marketing point to say you support the most number of digital assets that actually make things happen IRL.

Long live trustless tech. <3
